[2023-07-25 22:29:56,239] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: fetch_exchange_rates_dag.fetch_exchange_rates_task manual__2023-07-25T22:29:55.189208+00:00 [queued]>
[2023-07-25 22:29:56,249] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: fetch_exchange_rates_dag.fetch_exchange_rates_task manual__2023-07-25T22:29:55.189208+00:00 [queued]>
[2023-07-25 22:29:56,250] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-25 22:29:56,250] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-07-25 22:29:56,251] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-25 22:29:56,261] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): fetch_exchange_rates_task> on 2023-07-25 22:29:55.189208+00:00
[2023-07-25 22:29:56,267] {standard_task_runner.py:52} INFO - Started process 15001 to run task
[2023-07-25 22:29:56,269] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'fetch_exchange_rates_dag', 'fetch_exchange_rates_task', 'manual__2023-07-25T22:29:55.189208+00:00', '--job-id', '301', '--raw', '--subdir', 'DAGS_FOLDER/my_calc_rates.py', '--cfg-path', '/tmp/tmpdw72fus4', '--error-file', '/tmp/tmpek42hx18']
[2023-07-25 22:29:56,271] {standard_task_runner.py:80} INFO - Job 301: Subtask fetch_exchange_rates_task
[2023-07-25 22:29:56,334] {task_command.py:371} INFO - Running <TaskInstance: fetch_exchange_rates_dag.fetch_exchange_rates_task manual__2023-07-25T22:29:55.189208+00:00 [running]> on host 661c8d62a693
[2023-07-25 22:29:56,393] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=marina_z
AIRFLOW_CTX_DAG_ID=fetch_exchange_rates_dag
AIRFLOW_CTX_TASK_ID=fetch_exchange_rates_task
AIRFLOW_CTX_EXECUTION_DATE=2023-07-25T22:29:55.189208+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-07-25T22:29:55.189208+00:00
[2023-07-25 22:29:56,749] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/my_calc_rates.py", line 85, in fetch_exchange_rates
    cur.execute(create_table_query)
psycopg2.errors.SyntaxError: syntax error at or near "CREATE"
LINE 3:         CREATE TABLE exchange_rates_BTC (
                ^

[2023-07-25 22:29:56,759] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=fetch_exchange_rates_dag, task_id=fetch_exchange_rates_task, execution_date=20230725T222955, start_date=20230725T222956, end_date=20230725T222956
[2023-07-25 22:29:56,773] {standard_task_runner.py:97} ERROR - Failed to execute job 301 for task fetch_exchange_rates_task (syntax error at or near "CREATE"
LINE 3:         CREATE TABLE exchange_rates_BTC (
                ^
; 15001)
[2023-07-25 22:29:56,803] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-07-25 22:29:56,830] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
